### ISSUE TITLE
resource aws_autoscaling_lifecycle_hook doesn't support import

### ISSUE THREAD

**Issue by dvishniakov** [terraform-providers/terraform-provider-aws#9142]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Version
0.11.13

### Affected Resource(s)
* aws_autoscaling_lifecycle_hook
* aws_lambda_permission

### Terraform Configuration Files

```hcl
# Copy-paste your Terraform configurations here - for large Terraform configs,
# please use a service like Dropbox and share a link to the ZIP file. For
# security, you can also encrypt the files using our GPG public key: https://keybase.io/hashicorp
```

### Expected Behavior

Resources should be able importable
### Steps to Reproduce

1. `terraform apply`

**Comment by chaspy** [comment]
Hi, let me try.

**Comment by chaspy** [comment]
Implemented support for `aws_autoscaling_lifecycle_hook` https://github.com/terraform-providers/terraform-provider-aws/pull/9336 😄 

I'll implement for `aws_lambda_permission` too 👍

**Comment by bflad** [comment]
Support for importing `aws_autoscaling_lifecycle_hook` resources has been merged and will release with version 2.20.0 of the Terraform AWS Provider, expected later this week. 👍 

If you would like `aws_lambda_permission` resource import support, please submit that as a separate feature request.
